### ISSUE TITLE
I use the default params and get only 0.64 F1?

### ISSUE THREAD

**Issue by 1160300929** [monologg/NER-Multimodal-pytorch#3]
data_loader said 5151 words not in pretrained matrix ,is this normal??

**Comment by monologg** [comment]
Hi:)

1. `5151 words not in pretrained matrix` -> That is normal, even from the original paper's repository.
2. I've found out that I forgot to set seed before shuffling the data, and it makes issue on reproducibility.

**Comment by monologg** [comment]
![image](https://user-images.githubusercontent.com/28896432/82726761-395b7c80-9d21-11ea-9c7c-49643396dcc1.png)
I've add new commit to solve the reproducibility of the result. Please pull the new commit. You should run it on GPU to get the same result

**Comment by 1160300929** [comment]
Thanks!:)

**Comment by monologg** [comment]
@1160300929 
Also recommend you to see [this issue](https://github.com/jlfu/NERmultimodal/issues/9)!
I'll close this issue:) If you have any question, feel free to reopen the issue.
